Python/Product/TestAdapter/TestContainerDiscovererWorkspace.cs

```diff
@@ -288,5 +290,31 @@ class TestContainerDiscovererWorkspace : ITestContainerDiscoverer, IDisposable {
             }
             NotifyContainerChanged();
         }
+
+        private IEnumerable<string> GetTestFiles(ProjectInfo projectInfo) {
+            List<string> validFiles = Directory.EnumerateFiles(projectInfo.ProjectHome, "*.py", SearchOption.TopDirectoryOnly).ToList();
```

Comment:
Does validFiles.AddRange still work if we leave it as an IEnumerable?  ie. remove the ToList().

Reply:
There is no AddRange for IEnumerable, so I've replaced it with [IEnumerable.Concat()](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.concat?view=netframework-4.8).